sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -65,6 +70,28 @@
     "code.",
 ]
 
+_STANDARD_OPENTELEMETRY_HTTP_ATTRIBUTES = [
+    "client.address",
```

Comment:
Don't add client.* to above list because users could use client.foo as custom attribute with old sem conv.